mesheryctl/internal/cli/root/design/error.go

```diff
@@ -22,73 +22,81 @@ import (
 )
 
-func ErrPatternNotFound() error {
-	return errors.New(ErrPatternsNotFoundCode, errors.Fatal, []string{"Pattern Not Found"}, []string{"No Pattern found with the given name or ID"}, []string{"Pattern with the given name or ID is not present"}, []string{"Please check if the given pattern name or ID is present via 'mesheryctl pattern list'"})
+func ErrDesignNotFound() error {
+	return errors.New(ErrDesignsNotFoundCode, errors.Fatal, []string{"Design Not Found"}, []string{"No Design found with the given name or ID"}, []string{"Design with the given name or ID is not present"}, []string{"Please check if the given design name or ID is present via 'mesheryctl design list'"})
 }
 
-func ErrInvalidPatternFile(err error) error {
-	return errors.New(ErrInvalidPatternFileCode, errors.Fatal, []string{err.Error()}, []string{"Pattern appears invalid. Could not parse successfully"}, []string{"Pattern file provided is not valid"}, []string{"Please check that your pattern file is a valid yaml file"})
+func ErrInvalidDesignFile(err error) error {
+	return errors.New(ErrInvalidDesignFileCode, errors.Fatal, []string{err.Error()}, []string{"Design appears invalid. Could not parse successfully"}, []string{"Design file provided is not valid"}, []string{"Please check that your design file is a valid yaml file"})
 }
 
 func ErrPatternInvalidNameOrID(err error) error {
 	return errors.New(
-		ErrPatternInvalidNameOrIDCode,
+		ErrDesignInvalidNameOrIDCode,
 		errors.Alert,
-		[]string{"Unable to fetch Pattern"},
+		[]string{"Unable to fetch Design"},
 		[]string{err.Error()},
-		[]string{"Invalid pattern name or ID"},
-		[]string{"Run `mesheryctl pattern view -a` to view all available patterns."})
+		[]string{"Invalid design name or ID"},
+		[]string{"Run `mesheryctl design view -a` to view all available designs."})
 }
 
 func ErrImportPattern(err error) error {
-	return errors.New(ErrImportPatternCode, errors.Fatal,
-		[]string{"Unable to import pattern"},
+	return errors.New(ErrImportDesignCode, errors.Fatal,
+		[]string{"Unable to import design"},
 		[]string{err.Error()},
-		[]string{"Invalid pattern file"},
-		[]string{"Check your pattern URL/file path"})
+		[]string{"Invalid design file"},
+		[]string{"Check your design URL/file path"})
 }
 
 func ErrInValidSource(invalidSourceType string, validSourceTypes []string) error {
 	return errors.New(ErrInValidSourceCode, errors.Fatal,
-		[]string{fmt.Sprintf("Invalid pattern source type: `%s`", invalidSourceType)},
-		[]string{"Invalid pattern source type due to wrong type/passing"},
-		[]string{"pattern source type (-s) is invalid or not passed."},
+		[]string{fmt.Sprintf("Invalid design source type: `%s`", invalidSourceType)},
+		[]string{"Invalid design source type due to wrong type/passing"},
+		[]string{"design source type (-s) is invalid or not passed."},
 		[]string{"Ensure you pass a valid source type. \nAllowed source types: %s", strings.Join(validSourceTypes, ", ")})
 }
 
 func ErrPatternManifest() error {
```

Comment:
```suggestion
func ErrDesignManifest() error {
```
should it be updated ?